server/app.ts

```diff
@@ -58,7 +58,7 @@ export default function createApp(services: Services): express.Application {
   app.use(routes(services))
 
   app.use((req, res, next) => next(createError(404, 'Not found')))
-  app.use(errorHandler(process.env.NODE_ENV === 'production'))
+  app.use(errorHandler(services, config.production))
```

Comment:
In addition to passing services into the error handler middleware I've updated to use the `config.production` value here instead of reevaluating.